### PR TITLE
Internal: Add support link while opening an issue

### PR DESCRIPTION
**Pull request by sulunemre**
Issue template chooser menu items require a URL otherwise they are not shown to to the user.
This PR adds the missing URL for Support.

Here, `❓ Support Question` doesn't appear.
image

### CODE REVIEW

**Comment by sulunemre** [comment]
Hi @KingYes , this is a very trivial change and can be merged quickly.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 blank_issues_enabled: false
 contact_links:
   - name: ❓ Support Question
+    url: https://wordpress.org/support/plugin/elementor
```
Comment (KingYes) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://wordpress.org/support/plugin/elementor/
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
     about: Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!
   - name: Elementor Support
     url: https://wordpress.org/support/plugin/elementor
```
Comment (KingYes) [inline .github/ISSUE_TEMPLATE/config.yml:7]:
```suggestion
    url: https://elementor.com/support/
```

**Review by KingYes — CHANGES_REQUESTED** [review]

**Comment by KingYes** [comment]
Thanks!

**Pull request merged**